source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/index.tsx

```diff
@@ -2,5 +2,18 @@ import { connect } from 'react-redux';
 
 import HeaderDisplay from './display';
 
-//TODO: connect it to the notification store and logout action 
-export default connect(null, {})(HeaderDisplay);
\ No newline at end of file
+import {
+  toggleSidebar
+} from '@store/user-interface';
+
+const mapStateToProps = ({ ui }) => ({
+  isSidebarVisible: ui.isSidebarVisible
+});
+
+const mapDispatchToProps = {
```

Comment:
also, I'm _pretty sure_ that this needs to be the function form, because mapDispatchToProps needs to "map dispatch to props" when the functions within don't use dispatch themselves. example:

```ts
const mapDispatchToProps = dispatch => ({
  toggleSidebar: dispatch(toggleSidebar())
});
```

this way of writing mapDispatchToProps is needed because redux needs to know about the `toggleSidebar` object (returned from `toggleSidebar()` ), which contains the action `type` that redux uses to eventually uses to figure out which  reducer to run for that `type`